### ISSUE TITLE
Let users request deletion of images

### ISSUE THREAD

**Issue by bredowmax** [openfoodfacts/openfoodfacts-server#2891]
**Is your feature request related to a problem? Please describe.**
I just accidentally uploaded a wrong image. There is no indication of how I could undo my mistake.

**Describe the solution you'd like**
Mark an image as 'wrong' so that a moderator can safely remove it.

**Additional context**
https://world.openfoodfacts.org/product/0858335003100

**Comment by gkaklas** [comment]
I agree, but I believe a user should be able to delete an image by themselves, especially if they were the ones who uploaded it. Then, if an image was wrongfully deleted, a moderator could undo that change.

Also I think that the UX could prevent a user from saving a wrong image: as a new user, while editing I expected that the image would be saved when I saved my changes but it was saved immediately after I uploaded it.

**Comment by MonalikaPatnaik** [comment]
I think the issue has been resolved by adding the protection of images for normal users #8610 . So, as of now normal users can't upload/change the front photo of the product as it is hidden. However, we can do this for ingredients/nutrition/packaging image of the product maybe.

**Comment by Susexe** [comment]
I agree that a user should be able to delete photos that they uploaded themselves. For instance, I uploaded a duplicate image of ingredients, and now I don't even know whom to contact regarding this.

**Comment by alexgarel** [comment]
@Susexe we will try to add a "signal a photo" functionality soon.

**Comment by teolemon** [comment]
We now have Nutri-Patrol integration (http://github.com/openfoodfacts/nutripatrol/) but it's not easy to report an image just yet.

**Comment by JagjeevanAK** [comment]
@dev-rajm you can work on this issue no problem and show the draft PR.
